### PR TITLE
fix: consistent border-radius and button sizes in main layout header and sidebar

### PR DESCRIPTION
**Pull request by kopancek**
## Changes

Changes the border radius of buttons and button sizes in the main layout header, so that we are consistent across the board.

The figma design is here: https://www.figma.com/file/C7n8EiXBwV1sYIEHkQHS8R/daily.dev---Design-System?node-id=1886%3A8672&mode=dev

### Screenshots

Before:
Screenshot 2024-01-04 at 09 21 19
Screenshot 2024-01-04 at 09 21 12

After:
Screenshot 2024-01-04 at 15 37 01
Screenshot 2024-01-04 at 15 37 11
Screenshot 2024-01-04 at 15 47 20

### On those affected packages:
- [X] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [X] Does this not break anything in companion?

### Did you test the modified components media queries?
- [X] MobileL (420px)
- [X] Tablet (656px)
- [X] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
It looks so much better being consistent ✨

**Review by omBratteng — APPROVED** [review]

File: packages/shared/src/components/layout/MainLayoutHeader.tsx [hunk 1]
```diff
@@ -109,7 +108,7 @@ function MainLayoutHeader({
                   }
                 />
                 {hasNotification && (
-                  <Bubble className="right-0 top-0 -translate-y-1/2 translate-x-1/2 cursor-pointer px-1 shadow-bubble-cabbage">
+                  <Bubble className="-right-1.5 -top-1.5 cursor-pointer px-1">
```
Comment (rebelchris) [inline packages/shared/src/components/layout/MainLayoutHeader.tsx:111]: Did you mean to remove the `shadow-bubble-cabbage`?

File: packages/shared/src/components/post/write/CreatePostButton.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ export function CreatePostButton({
 }: CreatePostButtonProps): ReactElement {
   const { user, isAuthReady, squads } = useAuthContext();
   const { route, query } = useRouter();
-  const isTablet = useViewSize(ViewSize.Tablet);
+  const isMediumSize = useViewSize(ViewSize.Laptop);
```
Comment (rebelchris) [inline packages/shared/src/components/post/write/CreatePostButton.tsx:19]: This should only fire on Laptop and higher?
Maybe keep the naming to `isLaptop` in that case to be consistent in the app.

Although pretty sure tablet is a reverse one and targets laptop and higher already.

**Review by rebelchris — COMMENTED** [review]
One small comment, otherwise it does look complete to me.

File: packages/shared/src/components/post/write/CreatePostButton.tsx (hunk 2 above)
Comment (kopancek) [reply packages/shared/src/components/post/write/CreatePostButton.tsx:19]: This logic was completely opposite to what is in Figma -> show small size button on screens of smaller than laptop size (based on my understanding of the designs). I can change the variable name 👍

File: packages/shared/src/components/layout/MainLayoutHeader.tsx (hunk 1 above)
Comment (kopancek) [reply packages/shared/src/components/layout/MainLayoutHeader.tsx:111]: Yes, since there's no shadow in figma.

**Pull request merged**